src/api/profile/profile.api.js

```diff
@@ -72,6 +72,22 @@ async function getProfileDataForChunk(chunk, startingPoint, startingDist, output
                 )
             }
         } catch (err) {
+            if (
+                err.response &&
+                err.response.status === 413 &&
+                err.response.data.error?.message ===
+                    `Request Geometry contains too many points. Maximum number of points allowed: ${PROFILE_MAX_POINTS}, found ${chunk.coordinates.length}`
```

Comment:
Here simply check for the start of the message to avoid having the limit hardcoded